marketplace_front/components/pages/home/home.tsx

```diff
@@ -1,9 +1,22 @@
+import { PageTitle } from '../../molecules/page-title/page-title';
```

Comment:
```suggestion
import { PageTitle } from '../../molecules';
```

마이너한 건데 molecules/index.ts 써서 이렇게 처리하고 있었긴 합니다

Reply:
애초에 안 쓰이네요? 제거해주시면 될 거 같습니다!